Ginger/Ginger/ApplicationModelsLib/POMModels/POMEditPage.xaml.cs

```diff
@@ -96,6 +96,7 @@ public IWindowExplorer mWinExplorer
         public POMEditPage(ApplicationPOMModel POM, RepositoryItemPageViewMode editMode = RepositoryItemPageViewMode.View)
         {
             InitializeComponent();
+            xBusinessFlow.TargetApplication = POM.TargetApplicationKey.ItemName;
```

Comment:
this TargetApp set will break when user will change the selected Target Application after loading the Edit page.
so you also need to update it dynamicaly

Reply:
Done